### PR TITLE
rxvt-unicode 9.30

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
>   configure: error: unable to find libptytty

**Comment by chenrui333** [comment]
libptytty got spun off into a separate project, https://github.com/yusiwen/libptytty

**Comment by chenrui333** [comment]
revisit later.

File: Formula/rxvt-unicode.rb [hunk 1]
```diff
@@ -39,6 +45,12 @@ class RxvtUnicode < Formula
   end
 
   def install
+    resource("libptytty").stage do
+      system "cmake", ".", "-DCMAKE_INSTALL_PREFIX=#{libexec}", "-DBUILD_SHARED_LIBS=ON", *std_cmake_args
+      system "make", "install"
+    end
+    ENV.prepend_path "PATH", libexec/"lib"
```
Comment (chenrui333) [inline Formula/rxvt-unicode.rb:52]: need some help to understand the PATH setup for this resource.

File: Formula/rxvt-unicode.rb [hunk 2]
```diff
@@ -31,6 +32,11 @@ class RxvtUnicode < Formula
 
   uses_from_macos "perl"
 
+  resource "libptytty" do
+    url "https://github.com/yusiwen/libptytty/archive/b9694ea18e0dbd78213f55233a430325c13ad63e.tar.gz"
```
Comment (chenrui333) [inline Formula/rxvt-unicode.rb:36]: will wait for the tagged release in the upstream.

File: Formula/rxvt-unicode.rb (hunk 1 above)
Comment (carlocab) [reply Formula/rxvt-unicode.rb:52]: That doesn't look right. You probably want `libexec/"bin"`.

File: Formula/rxvt-unicode.rb [hunk 3]
```diff
@@ -39,6 +45,12 @@ class RxvtUnicode < Formula
   end
 
   def install
+    resource("libptytty").stage do
+      system "cmake", ".", "-DCMAKE_INSTALL_PREFIX=#{libexec}", "-DBUILD_SHARED_LIBS=ON", *std_cmake_args
+      system "make", "install"
```
Comment (carlocab) [inline Formula/rxvt-unicode.rb:50]:
```suggestion
      system "cmake", "-S", ".", "-B", "build", "-DBUILD_SHARED_LIBS=ON", *std_cmake_args(install_prefix: libexec)
      system "cmake", "--build", "build"
      system "cmake", "--install", "build"
```

File: Formula/rxvt-unicode.rb (hunk 2 above)
Comment (carlocab) [reply Formula/rxvt-unicode.rb:36]: We should consider making this its own formula if it gets a release tag.

Comment (chenrui333) [reply Formula/rxvt-unicode.rb:36]: Agree.

**Pull request closed without merging**